File: d12Mon/C#/program/p00check_id/CheckId.cs
namespace p00check_id
{
  class CheckId{
    public string id;
    public string pw;
    public CheckId(string id, string pw){
      this.id = id;
      this.pw = pw;
    }
    public void checkId(){
      if(this.id == "jin" && this.pw == "111"){
        Console.WriteLine("yes");
      } else{
        Console.WriteLine("no");
      }
    }
  }
}

File: d12Mon/Php/program/p00check_id/CheckId.php
<?php
class CheckId{
  private string $id;
  private string $pw;
  public function __construct(string $id, string $pw){
    $this->id = $id;
    $this->pw = $pw;
  }
  public function checkId(){
    if($this->id == "jin" && $this->pw == "111"){
      echo "yyy";
    } else{
      echo "nnn";
    }
  }
}